docs/developers/coredev/release.md

```diff
@@ -2,52 +2,55 @@
 
 # Release guide
 
-This guide documents `napari`'s release process.
-Currently, it only handles distribution, but as the project matures,
-it will include generating release notes, documentation, etc.
+This guide documents `napari`'s release process. Most required tools mentioned here are in https://github.com/napari/napari-release-tools
 
 # Timeline
-New versions of `napari` will be released every two months. The first release candidate will be available one week prior to release for testing purposes. Multiple release candidates may become available during the week prior to release. Upcoming releases can be found in our public calendar.
+
+Currently, we are releasing a new napari version every 2-3 weeks. Release candidates are made available as a reasonableness check for 1-24h before the full release is published: since the release cadence is so high, any new bugs can be quickly rectified, and more quickly identified by making a full release.
+
+When major API changes and deprecations happen, we will provide longer release candidate cycles to ensure proper testing by the community.
 
 The latest release candidate can be installed with
 
 `python -m pip install --pre napari`
 
 # Release management
-The release will be coordinated by a release manager whose responsibilities include...
+The release will be coordinated by a release manager whose responsibilities include the following:
 
 ## Two weeks before release (one week before release candidate)
```

Comment:
probably worth updating the timeline given our current cadence. I would say:

- one week before release: look through open PRs etc
- 1-2 days before release: generate release notes
- 1-0 days before release: make rc
- day of release: make sure rc is tested and docs are deployed, then release